viadot/sources/epicor.py

```diff
@@ -232,7 +234,7 @@ def generate_token(self) -> str:
             "password": self.credentials["password"],
         }
 
-        response = requests.request("POST", url, headers=headers)
+        response = handle_api_response(url=url, headers=headers, method="post")
```

Comment:
Let's keep the convention for names of HTTP methods and capitalize them:
`get` -> `GET`
`post` -> `POST`